src/Catalog/Dnx/DnxMaker.cs

```diff
@@ -27,13 +27,15 @@ public DnxMaker(StorageFactory storageFactory)
             _storageFactory = storageFactory;
         }
 
-        public async Task AddPackage(Stream nupkgStream, string nuspec, string id, string version, CancellationToken cancellationToken)
+        public async Task<Tuple<Uri, Uri>> AddPackage(Stream nupkgStream, string nuspec, string id, string version, CancellationToken cancellationToken)
         {
             Storage storage = _storageFactory.Create(id);
 
-            await SaveNuspec(storage, id, version, nuspec, cancellationToken);
-            await SaveNupkg(nupkgStream, storage, id, version, cancellationToken);
+            var nuspecUri = await SaveNuspec(storage, id, version, nuspec, cancellationToken);
+            var nupkgUri = await SaveNupkg(nupkgStream, storage, id, version, cancellationToken);
             await UpdateMetadata(storage, versions => versions.Add(NuGetVersion.Parse(version)), cancellationToken);
+
+            return new Tuple<Uri, Uri>(nuspecUri, nupkgUri);
```

Comment:
Fixed

---
In reply to: [57124124](https://github.com/NuGet/NuGet.Services.Metadata/pull/60#discussion_r57124124) [](ancestors = 57124124)